lib/logstasher/railtie.rb

```diff
@@ -41,6 +41,20 @@ class Railtie < Rails::Railtie
     end
   end
 
+  def default_source
+    case RUBY_PLATFORM
+    when /darwin/
+      # NOTE: MacOS Sierra and later are setting `.local`
```

Comment:
**10.2**, not _10.12_! Later means  _after_. 

There is no later version than Sierra! The issue occurs on version of OS X starting with OS X Jaguar. This is way back in 2002, when Apple introduced the Bonjour service to the platform (2005 if you start counting from the renaming of the service).

I'm far from thinking someone will use OS X that old, but you get the point (at least I hope). You *will* have the same issue with El Capitan, Yosemite, Mavericks and so on...